src/pages/index.mdx

```diff
@@ -1,3 +1,41 @@
-# Fabric Design System
+<div className="grid grid-cols-3 gap-16">
```

Comment:
The content should wrap on smaller devices, right now it looks a bit weird 
![image](https://user-images.githubusercontent.com/16625915/135491047-cea946f8-3079-42b7-be6c-31ddd4d7b067.png)
![image](https://user-images.githubusercontent.com/16625915/135491065-54e89957-93ac-4304-9828-efef03470403.png)

Reply:
Yes, thanks. Ill plan to do trim responsive scaling when the layout is desided upon ;)

Reply:
![image](https://user-images.githubusercontent.com/462825/135528278-d254689a-9d84-438b-aaa9-ed9afad29374.png)
![image](https://user-images.githubusercontent.com/462825/135528330-43836fbe-a78e-44e8-8907-defd1d438004.png)

Should work now.